etc/kayobe/ansible/ubuntu-upgrade.yml

```diff
@@ -0,0 +1,112 @@
+---
+# To prevent Ansible role dependency errors, this playbook requires that environment variable
+# ANSIBLE_ROLES_PATH is defined and includes '$KAYOBE_PATH/ansible/roles' at seed-hypervisor.
+- name: Migrate hosts from Ubuntu Focal 20.04 to Jammy 22.04
+  hosts: overcloud:infra-vms:seed
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Assert that hosts are running Ubuntu Focal
+      assert:
+        that:
+          - ansible_facts.distribution == 'Ubuntu'
+          - ansible_facts.distribution_major_version == '20'
+          - ansible_facts.distribution_release == 'focal'
```

Comment:
NIT: Not sure both of these are needed

Reply:
We wanted to be extra sure about this assert.

Reply:
Ubuntu 20 is always going to be Focal 😄 

But either way looks fine. Lets get this moving 👍